examples/notebooks/stationarity_detrending_0.ipynb

```diff
@@ -0,0 +1,304 @@
+   "source": [
+    "## ADF test\n",
+    "\n",
+    "ADF test is used to determine the presence of unit root in the series, and hence helps in understand if the series is stationary or not. The null and alternate hypothesis of this test are:\n",
+    "\n",
+    "Null Hypothesis: The series has a unit root (value of a =1).\n",
+    "\n",
+    "Alternate Hypothesis: The series has no unit root.\n",
+    "\n",
+    "If the null hypothesis in not rejected, the series is non-stationary.\n",
+    "\n",
+    "A function is created to test carry out the ADF test on a time series."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from statsmodels.tsa.stattools import adfuller\n",
+    "def adf_test(timeseries):\n",
+    "    print ('Results of Dickey-Fuller Test:')\n",
+    "    dftest = adfuller(timeseries, autolag='AIC')\n",
+    "    dfoutput = pd.Series(dftest[0:4], index=['Test Statistic','p-value','#Lags Used','Number of Observations Used'])\n",
+    "    for key,value in dftest[4].items():\n",
+    "       dfoutput['Critical Value (%s)'%key] = value\n",
+    "    print (dfoutput)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## KPSS test\n",
+    "\n",
+    "KPSS is another test for checking the stationarity of a time series. The null and alternate hypothesis for the KPSS test are opposite that of the ADF test.\n",
+    "\n",
+    "Null Hypothesis: The process is trend stationary.\n",
+    "\n",
+    "Alternate Hypothesis: The series has a unit root (series is not stationary).\n",
+    "\n",
+    "A function is created to test carry out the KPSS test on a time series."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from statsmodels.tsa.stattools import kpss\n",
+    "def kpss_test(timeseries):\n",
+    "    print ('Results of KPSS Test:')\n",
+    "    kpsstest = kpss(timeseries, regression='c', nlags=None)\n",
+    "    kpss_output = pd.Series(kpsstest[0:3], index=['Test Statistic','p-value','Lags Used'])\n",
+    "    for key,value in kpsstest[3].items():\n",
+    "        kpss_output['Critical Value (%s)'%key] = value\n",
+    "    print (kpss_output)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The ADF tests gives the following results – test statistic, p value and the critical value at 1%, 5% , and 10% confidence intervals.\n",
+    "\n",
+    "ADF test is now applied on the data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "adf_test(sunspots['SUNACTIVITY'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Critical value of p is assumed to be 0.05. Based upon the p-value of ADF test, the null hypothesis can not be rejected. Hence, the series is non-stationary."
```

Comment:
I'm not sure what you mean by "assumed to be 0.05", but again I would prefer if the statement was rewritten similar to what I wrote above.

Reply:
I meant the significance level. I shall reword it.